app/views/dashboards/show.html.erb

```diff
@@ -24,6 +24,18 @@
             <span class="label label-info" id="pull-requests-count">
               <%= pull_requests.count %>
             </span>
+            <% if pull_requests.count == 1 %>
+              <% tweetstring = t("dashboard.tweet_progress_singular") %>
+            <% else %>
+              <% tweetstring = t("dashboard.tweet_progress_plural", count: pull_requests.count) %>
+            <% end %>
```

Comment:
You don;t need to add logic for single and plural. See how to do this properly: https://robots.thoughtbot.com/pluralizing-rails-i18n-translations

Reply:
Wow - TIL. Well done, Rails.

Reply:
<3 Rails!